pyteal/ast/subroutine.py

```diff
@@ -612,10 +613,33 @@ def method_signature(self, overriding_name: str = None) -> str:
     def method_spec(self) -> sdk_abi.Method:
         skip_names = ["return", "output"]
 
+        desc: str = ""
+        arg_descs: dict[str, str] = {}
+        output_desc: str = ""
+
+        if self.subroutine.implementation.__doc__ is not None:
+            docstring = parse_docstring(self.subroutine.implementation.__doc__)
```

Comment:
another problem: you do not know what style of docstring gets passed in, and `parse` can't decide on its own. Seems we don't have ability to tell which style of docstring just from `docstring_parser` lib, and wonder if we can correctly support 4 styles of docstrings.

Reply:
The `docstring` var should contain an element that specifies what type of docstring it thinks it is, or at least it did in the one example i tried marking it as `Google`